server/db/schema.sql

```diff
@@ -7,4 +8,21 @@ CREATE TABLE pet_types (
   description TEXT
 );
 
+CREATE TABLE adoptable_pets (
+  id SERIAL PRIMARY KEY,
+  name VARCHAR(255) NOT NULL,
+  img_url VARCHAR(255) NOT NULL,
+  age INTEGER, 
+  vaccination_status BOOLEAN NOT NULL,
+  adoption_story TEXT NOT NULL,
+  available_for_adoption BOOLEAN NOT NULL,
+  pet_type_id INTEGER REFERENCES pet_types(id)
+);
+
+ALTER TABLE adoptable_pets
+ALTER COLUMN vaccination_status
+SET DEFAULT FALSE;
 
+ALTER TABLE adoptable_pets
+ALTER COLUMN available_for_adoption
+SET DEFAULT TRUE;
```

Comment:
These defaults can be set within the CREATE TABLE. If we were creating migrations (we'll learn about soon!) we would want instructions like this to help build our database with changes over time. In our case here the schema will be run once to create the database, so all set up can be declared in the CREATE TABLEs rather than separate commands to further modify the table after it's been made